@navikt/core/css/darkside/form/form-summary.darkside.css

```
@@ -22,7 +22,6 @@
 
 .aksel-form-summary__edit {
```

Ooh, we might want to do this
```suggestion
.aksel-form-summary__header .aksel-form-summary__edit {
```
(Then we know we won't break anything in the footer when removing it later.)

---

Done 🏁